src/explore-education-statistics-common/src/services/types/methodology.ts

```diff
@@ -1,7 +1,13 @@
+export type MethodologyApprovalStatus =
+  | 'Draft'
+  | 'HigherLevelReview'
+  | 'Approved';
```

Comment:
We shouldn't need to this in common as it's admin specific. I think this correlates with another comment by @benoutram about exposing these details to the public frontend.

Would suggest we revert all the changes around moving the status into common.